examples/src/main/java/com/vesoft/nebula/examples/MetaClientExample.java

```diff
@@ -14,8 +14,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(MetaClientExample.class);
 
     public static void main(String[] args) {
-        MetaClientImpl metaClient = new MetaClientImpl("127.0.0.1", 45500);
-        LOGGER.info(metaClient.getPart(2, 1).toString());
+        MetaClientImpl metaClient = new MetaClientImpl("127.0.0.1", 28910);
```

Comment:
the default port of meta service is 45500 please use the default value in examples